region/wreckedship/main.json

```diff
@@ -807,6 +807,9 @@
                 {"canShineCharge": {
                   "usedTiles": 31
                 }},
+                {"cost": [
+                  {"shinesparkFrames": 40}
```

Comment:
When we talked about it, I thought you preferred putting the shinespark  costs inside the `canShineCharge` and `canComeInCharged` objects. Did you change your mind?

Reply:
No but it didn't feel like we had come to a definite agreement about it. Guess we did ^^